### PR TITLE
use tape-style assertions inside selenium

### PR DESCRIPTION
**Pull request by fippo**
'cause @jan-ivar showed how. Still looking for a way to not duplicate the definition of t in the browser environment...

### CODE REVIEW

File: test/test.js [hunk 1]
```diff
@@ -1231,7 +1231,13 @@ test('Basic connection establishment', function(t) {
       },
       is: function(a, b, msg) {
         this.ok((a === b), msg + ' - got ' + b);
-      }
+      },
+      pass: function(msg) {
+        window.testPassed.push(msg);
```
Comment (jan-ivar) [inline test/test.js]: or `this.ok(true, msg);`

File: test/test.js [hunk 2]
```diff
@@ -1231,7 +1231,13 @@ test('Basic connection establishment', function(t) {
...
+      },
+      fail: function(msg) {
+        window.testFailed.push(msg);
```
Comment (jan-ivar) [inline test/test.js]: or `this.ok(false, msg);`

File: test/test.js (hunk 1 above)
Comment (fippo) [inline test/test.js]: even better!

File: test/test.js [hunk 3]
```diff
@@ -1285,64 +1291,56 @@ test('Basic connection establishment', function(t) {
 
       pc1.createOffer(
         function(offer) {
-          window.testPassed.push('pc1.createOffer');
+          t.ok('pc1.createOffer');
```
Comment (jan-ivar) [inline test/test.js]: I think you mean `t.pass` here (and below).

**Comment by fippo** [comment]
note: it would be good to also "shim" t.plan here which might have helped catching #179. Not sure what the best way to do it is though

**Comment by KaptenJansson** [comment]
LGTM.

Not sure you can use t.plan() here anyway due to having t.pass on the onicecandidate event (we do not know how many candidates will be generated), this could ofc be changed to only trigger once which would make t.plan() usable. But I think it's fine as it is for now.

**Pull request merged**